protos/flyteidl/plugins/sagemaker/parameter_ranges.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package flyteidl.plugins.sagemaker;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
+
+// HyperparameterScalingType defines the way to increase or decrease the value of the hyperparameter
+// https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-define-ranges.html
+enum HyperparameterScalingType {
```

Comment:
You probably want to wrap this with a message... otherwise I think the enum name (`HyperparameterScalingType`) will be dropped in some of the generated languages...

```suggestion
message HyperparameterScalingType {
   enum Values {
   
```

Reply:
but again, double check the other enums we have :-)